AsyncSwift/Models/JSONResponse.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  EventResponse.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/08.
+//
+
+import Foundation
+
+struct JSONResponse: Codable {
+
+    init() {
+        self.event = Event()
+        self.sessions = []
+    }
+    var event: Event
+    var sessions: [Session]
```

Comment:
```suggestion
    var event = Event()
    var sessions: [Session] = []
```
property에 값이 init에서 무조건 주입된다면, init method가 없어도 되지 않을까 싶네요.. !

Reply:
확인했습니다 수정하도록 하겠습니다